test/tracer_test.rb

```diff
@@ -170,7 +170,10 @@ def test_trace_all_args
     assert_equal('extra-resource', span.resource)
     assert_equal('my-type', span.span_type)
     assert_equal(yesterday, span.start_time)
-    assert_equal({ 'env' => 'test', 'temp' => 'cool', 'tag1' => 'value1', 'tag2' => 'value2' }, span.meta)
+    assert_equal('test', span.get_tag('env'))
+    assert_equal('cool', span.get_tag('temp'))
+    assert_equal('value1', span.get_tag('tag1'))
+    assert_equal('value2', span.get_tag('tag2'))
```

Comment:
I left it out on purpose. I have mixed feelings about it. On one hand it's nice to check more tags haven't been added, on the other hand it means that adding automatic tags like the pid here means having to update unrelated tests. If you think that the pros outweigh the cons, I'll add that check.